src/main/java/essenmakanan/recipe/RecipeList.java

```diff
@@ -32,4 +35,38 @@ public Recipe getRecipe(int index) {
         return recipes.get(index);
     }
 
+    public void viewSpecificRecipe(int index) {
+        Ui.drawDivider();
+        if (index < 0 | index >= recipes.size()) {
```

Comment:
i think the or symbol is "||", one line is for bitwise

Reply:
modified! Thanks for the notice